go-apps/meep-app-enablement/server/app-info/app-info.go

```diff
@@ -362,6 +364,17 @@ func applicationsAppInstanceIdDELETE(w http.ResponseWriter, r *http.Request) {
 
 	fields, err := rc.GetEntry(appEnablementBaseKey + ":apps:" + appInstanceId)
 	if err == nil && len(fields) > 0 {
+
+		//deletes all the services related to that app instance ID
+		err = msmgmt.AppServicesDELETE(appInstanceId)
```

Comment:
Why is this done before the app instance termination confirmation / timeout?

Reply:
termination not handled yet... a task was opened for that part...